ui/src/components/scroll-area/QScrollArea.js

```diff
@@ -154,97 +244,162 @@ export default Vue.extend({
       return this.$refs.target
     },
 
+    getScroll () {
+      const info = {}
+
+      axisList.forEach(axis => {
+        const data = this.scrollComputed[ axis ]
+
+        info[ axis + 'Position' ] = data.position
+        info[ axis + 'Percentage' ] = data.percentage
+        info[ axis + 'Size' ] = data.size
+        info[ axis + 'ContainerSize' ] = this.container[ axis ]
+      })
+
+      return info
+    },
+
     getScrollPosition () {
-      return this.scrollPosition
+      return {
+        top: this.scroll.vertical.position,
+        left: this.scroll.horizontal.position
+      }
     },
 
-    setScrollPosition (offset, duration) {
-      const fn = this.horizontal === true
-        ? setHorizontalScrollPosition
-        : setScrollPosition
+    getScrollPercentage () {
+      return {
+        top: this.verticalScrollPercentage,
+        left: this.horizontalScrollPercentage
+      }
+    },
+
+    setScrollPosition (axis, offset, duration) {
```

Comment:
Put the `axis` as third parameter so we don't do breaking changes pls.

Reply:
I'll check first parameter if it's vertical/horizontal and change params